src/olympia/addons/models.py

```diff
@@ -1775,6 +1776,21 @@ def set_tag_list(self, new_tag_list):
                 tag.remove_tag(self)
         self.tag_list = new_tag_list
 
+    def update_all_due_dates(self):
+        for version in Version.objects.should_have_due_date().filter(
```

Comment:
Can use `self.versions` instead of `Version.objects.(...).filter(addon=self)` here and below